### PR TITLE
Setup solium and lint contracts

### PR DESCRIPTION
**Pull request by Ryan-Gordon**
Linted the files with solium. Also added 2 scripts to package json. One for linting and reporting errors, one for live linting as we write contracts.
Do we need solium as a dev dep in the package.json? I have it installed globally.

Heres an image of the warnings left behind after linting.

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (PaddyMc) [inline package.json]: Awesome work man!

Can we add solium as a devDependency, as you mentioned in the MR description, please? And then I'm very happy to have this merged in :heart:

File: .soliumrc.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (PaddyMc) [inline .soliumrc.json]: Also if you want to add this to our `.soliumrc.json`, that'd be ace? 

```
{
  "extends": "solium:all",
  "plugins": ["security"],
  "rules": {
    "arg-overflow": "error",
    "blank-lines": "error",
    "camelcase": "error",
    "deprecated-suicide": "error",
    "emit": "error",
    "error-reason": "off",
    "function-order": "off",
    "indentation": ["error", 2],
    "linebreak-style": ["error", "unix"],
    "no-constant": "error",
    "no-empty-blocks": "error",
    "pragma-on-top": "error",
    "quotes": ["error", "double"],
    "uppercase": "off",
    "visibility-first": "error",
    "whitespace": "error",

    "security/enforce-loop-bounds": "warning",
    "security/no-assign-params": "off",
    "security/no-func-overriding": "error",
    "security/no-inline-assembly": "off",
    "security/no-low-level-calls": "off",
    "security/no-suicide-or-selfdestruct": "error",
    "security/no-var": "error"
  }
}

```

:gem:

Comment (PaddyMc) [reply .soliumrc.json]: :arrow_up: if this throws loadsa errors, just push merge and I'll solve the conflicts :+1:

**Comment by Ryan-Gordon** [comment]
Updated the `.soliumrc.json` and fixed the extra errors 👍

**Pull request merged**